webpack/build.js

```diff
@@ -11,9 +11,8 @@ module.exports = function(webpackConfig) {
       var jsonStats = stats.toJson()
       var buildError = fatalError || jsonStats.errors[0] || jsonStats.warnings[0]
 
-      if (buildError) {
+      if (buildError)
```

Comment:
Either there was errors in existing code that bracket was missing, or bracket was unnecessary in one-line blocks. I chosen setup with least errors in existing code. It is impossible to setup linter to not warn on "redundant" brackets.